core/templates/dev/head/services/contextual/WindowDimensionsService.ts

```diff
@@ -16,26 +16,37 @@
  * @fileoverview Service for computing the window dimensions.
  */
 
-angular.module('oppia').factory('WindowDimensionsService', [
-  '$window', function($window) {
-    var onResizeHooks = [];
-    angular.element($window).bind('resize', function() {
-      onResizeHooks.forEach(function(hookFn) {
-        hookFn();
-      });
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { HostListener, Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WindowDimensionsService {
+  onResizeHooks : Array<any> = [];
+
+  @HostListener('window:resize', ['$event'])
+  onResize(event) {
```

Comment:
I think you missed this.

Reply:
It logically returns nothing but in javascript I guess if the function is a single line function it automatically returns the first line but the return type should be void right?

Reply:
Yup. (Also do check if the typescript compiles fine -- `python -m scripts.typescript_checks`)

Reply:
@bansalnitish Yup it's successful!
![Screenshot 2019-10-13 at 9 53 57 PM](https://user-images.githubusercontent.com/30312043/66718677-043e9180-ee04-11e9-9533-1bb7e858bf9f.png)

Reply:
Done